managers/migrations/0001_initial.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 2.1.1 on 2018-11-13 12:05
+
+from django.conf import settings
+from django.db import migrations, models
```

Comment:
Split into two separate lines.

Reply:
But I didn't write migrations...

I did not write it, it's not true!
It's bullshit! I did not write it!
I did not!
Oh hi, @maciejSamerdak !

Reply:
Oh hi @Karrp, what's up?

Reply:
It's about @TheCM, he says I wrote migrations.

Reply:
What?! Well did you?

Reply:
No! It's not true! Don't even ask!

Reply:
U R actually ->